evansmusomi/react-native-102
# Locates User on a Map

*What this does*
- Locates user on a map

*How to verify*
- Turn on your Location Settings, if using a real Android Device
- Navigate to the Share Place screen
- Press the Locate Me button
- The map should animate to your location and place a marker there
